sw/device/lib/runtime/otbn.c

```diff
@@ -190,3 +190,24 @@ otbn_result_t otbn_dump_dmem(const otbn_t *ctx, uint32_t max_addr) {
 
   return kOtbnOk;
 }
+
+otbn_result_t otbn_zero_instruction_memory(otbn_t *ctx) {
+  if (ctx == NULL) {
+    return kOtbnBadArg;
+  }
+
+  size_t imem_size_words =
+      dif_otbn_get_imem_size_bytes(&ctx->dif) / sizeof(uint32_t);
+  bool retval = kOtbnOk;
+  for (size_t i = 0; i < imem_size_words; ++i) {
```

Comment:
Why not use `OTBN_IMEM_SIZE_WORDS` here?

Reply:
I've just copied the this function from `otbn_zero_dmem()` or similar. I don't remember the exact name. Anyway, I've now removed it because it's no longer needed as we can initialize the memories at simulation start (VCS/Xcelium only, it doesn't matter for Verilator and FPGA).